### PR TITLE
HW-1: Add base HTML and CSS

### PR DESCRIPTION
**Pull request by rogerosha**
This pull request adds a web page showcasing my personal introduction and information, inspired by the provided Pokemon design. It's built using HTML and CSS, creating an engaging and personalized online presence.

image

### CODE REVIEW

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            type="text/css">
+    </head>
+    <body>
+        <h1>About me</h1>
+        <p>
+            Hi! My name is Katya and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like
+            HTML, CSS, JavaScript, and Git version control system.
+        </p>
+        <p>
+            This page was developed during the course <a href="https://www.mastersacademy.education" class="colortext"> 'Frontend for beginners' from Masters Academy in 2023</a>.
```
Comment (vladyslav-yermolin-moc) [inline homeworks/ekaterina.sharova_rogerosha/PokemonProject/index.html:21]: Hey, @rogerosha, please find a decent name for the link class name, e.g. `link`)))

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.colortext {
```
Comment (vladyslav-yermolin-moc) [inline homeworks/ekaterina.sharova_rogerosha/PokemonProject/style.css:1]: Please add text properties (`font-size`, `line-heght`, `font-weight`, `color`,...) from the design to link, header, pragraph
image

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

**Comment by rogerosha** [comment]
Fixed the issues! :) image

**Comment by rogerosha** [comment]
Fixed the issues

image

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**